ppa/archive/admin.py

```diff
@@ -17,6 +18,14 @@ class DigitizedWorkAdmin(admin.ModelAdmin):
     list_filter = ['collections']
     actions = ['bulk_add_collection']
 
+    def list_collections(self, obj):
```

Comment:
I might have put the list property on the model rather than here, but I don't know that we would use it anywhere else.  Doesn't seem like any point in trying to share logic with the collection list we use for index data.

Reply:
@rlskoeser That was my thought. I initially had it on the model and then decided that this was a once off use, unless you can think of reasons otherwise.